stable/node-problem-detector/README.md

```diff
@@ -1,42 +1,16 @@
 # node-problem-detector
 
-![Version: 2.0.3](https://img.shields.io/badge/Version-2.0.3-informational?style=flat-square) ![AppVersion: v0.8.9](https://img.shields.io/badge/AppVersion-v0.8.9-informational?style=flat-square)
+![Version: 2.0.4](https://img.shields.io/badge/Version-2.0.4-informational?style=flat-square) ![AppVersion: v0.8.9](https://img.shields.io/badge/AppVersion-v0.8.9-informational?style=flat-square)
 
 This chart installs a [node-problem-detector](https://github.com/kubernetes/node-problem-detector) daemonset. This tool aims to make various node problems visible to the upstream layers in cluster management stack. It is a daemon which runs on each node, detects node problems and reports them to apiserver.
 
 **Homepage:** <https://github.com/kubernetes/node-problem-detector>
 
-## How to install this chart
```

Comment:
Why is all this being removed?

Reply:
No idea; I generated the README according to the contribution instructions, I assumed that the behavior was intended.

Reply:
I see now -- the location from which helm-docs is run from matters. The REAME looks more sane now.

Reply:
Thanks!